File: funtions/notes.py
# Alan De Lara, Funtions Notes Python

#Funtinos hold actions to be reused
#number = int(input("Please tell me a number:\n"))
#def add(numOne, numTwo): # paramneters set the name of the varible (just for the funtion)
    
   # return numOne + numTwo 
#print(add(number,21))#arguemnets set the value of the varible just for that instacen of the futnion.
#print(add(8,12))
#addition = add(6,4)
#print(add(addition,10000))
#add(87,3)

def values(type):
    return input(f"Please give me a {type}:\n")

name = values ("name")
place =values ("place")
verb =values ("verb (past tense)")

print(f"{name} was rlly fast getting to {place} becase the {verb} the whole way there.")

File: funtions/notes.c
//Alan De Lara. Funtion Notes
#include <stdio.h>
int num;
char name[50], place[50], verb[50];
int add(int numOne, int numTwo){
     return numOne+numTwo;
}
void due(char assigment[50], char day[20]){
    printf("the %s assigmnet is due %s\n", assigment, day);
}
int main(void){
    //printf("Please tell me a number:\n");
    //scanf("%d", num);
    //add(num,10);
    //add(8,1);
    //add(4,85);
due ("funtions notes, Today");
due ("Hello world Update,  due tommorow");
due ("fanceil caltor Update, due friday");

    return 0;
}